js/app.js

```diff
@@ -1,30 +1,23 @@
-// wait for DOM to load before running JS
 $(document).ready( function() {
-
-  // check to make sure JS is loaded
   console.log('JS is loaded!');
-//var url_link="https://developers.zomato.com/api/v2.1/cities";
-var url_link="https://api.tvmaze.com/shows?q=";
```

Comment:
You do not need the `?q=` at the end of this link. When you serialize the form using `$('form').serialize()`, it adds `?q=yourSearchTerm` to the end of the URL.

That is because in your HTML, that search input field has `name="q"` as an attribute.